bin/biotype_table_parser.r

```diff
@@ -3,13 +3,13 @@
 library(dplyr)
 library(data.table)
 table1= data.frame(matrix(ncol = 66, nrow = 0))
```

Comment:
The  `biotype_table_parser.r` script is long overdue for a change. It has several hard-coded parameters that should not be that way: https://github.com/Flomics/rnaseq/issues/62. If the script tries to initialize a 0x0 data frame (by `table1 <- data.frame()`  and then run the following line to name the columns, the following error is thrown: 
```
  'names' attribute [66] must be the same length as the vector [0]
```
So this specific issue of hardcoding the number of columns would be fixed if we did not hard-code the names of the biotypes in the script. 

In any case, this script is only needed for the "legacy" biotype assignment. We can find a better solution to the hard-coding issues for the time we keep the legacy assignment, but it will become obsolete once we remove the legacy mode.